docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -0,0 +1,281 @@
+---
+title: Pod Lifecycle
+---
+
+{% capture overview %}
+
+{% comment %}Updated: 4/14/2015{% endcomment %}
+{% comment %}Edited and moved to Concepts section: 2/2/17{% endcomment %}
+
+This page describes the lifecycle of a Pod.
+
+{% endcapture %}
+
+
+{% capture body %}
+
+## Pod phase
+
+A Pod's `status` field is a
+[PodStatus](/docs/resources-reference/v1.5/#podstatus-v1)
+object, which has a `phase` field.
+
+Pod phases are consistent with the overall Kubernetes
+[API convention](https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md).
+
+The phase of a Pod is a simple, high-level summary of where the Pod is in its
+lifecycle. The phase is not intended to be a comprehensive rollup of observations
+of Container or Pod state, nor is it intended to be a comprehensive state machine.
+
+The number and meanings of Pod phase values are tightly guarded.
+Other than what is documented here, nothing should be assumed about Pods that
+have a given `phase` value.
+
+Here are the possible values for `phase`:
+
+* Pending: The Pod has been accepted by the Kubernetes system, but one or more of
+  the Container images has not been created. This includes time before being
+  scheduled as well as time spent downloading images over the network,
+  which could take a while.
+
+* Running: The Pod has been bound to a node, and all of the Containers have been
+  created. At least one Container is still running, or is in the process of
+  starting or restarting.
+
+* Succeeded: All Containers in the Pod have terminated in success, and will not
+  be restarted.
+
+* Failed: All Containers in the Pod have terminated, and at least one Container
+  has terminated in failure. That is, the Container either exited with non-zero
+  status or was terminated by the system.
+
+* Unknown: For some reason the state of the Pod could not be obtained, typically
+  due to an error in communicating with the host of the Pod.
+
+## Pod conditions
+
+If any Container in a Pod specifies a readiness probe, the Pod reports a
+[PodCondition](/docs/resources-reference/v1.5/#podcondition-v1).
+A PodCondition has a `status` field that can be True, False, or Unknown.
+
+## Container probes
+
+A [Probe](/docs/resources-reference/v1.5/#probe-v1) is a diagnostic
+performed periodically by the kubelet on a Container. To perform a diagnostic,
```

Comment:
Going forward, our [style](https://kubernetes.io/docs/contribute/style-guide/#use-camel-case-for-api-objects) is to use the same case as the API object: Container, PodSpec, etc.